index.html

```diff
@@ -166,6 +167,7 @@ <h2>Uber</h2>
   </section>
 
 
+
   <!--mobile cards-->
   <section id="portfolio" class="grid-container">
```

Comment:
- [ ] <b>REQUIRED:</b> In HTML, content of Mobile and Desktop screens should not be different. that means you don't have to duplicate HTML codes in other to make your page responsive. This can be taken care of by just CSS. Read more about CSS [MEDIA QUERIES](https://www.w3schools.com/css/css3_mediaqueries.asp)

Reply:
Can you explain more please, I didn't really understand what I have to do.